jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaData.java

```diff
@@ -467,15 +468,13 @@ public String getSchemaTerm() {
   }
 
   @Override
-  public ResultSet getSchemas() {
-    LOGGER.debug("Schemas are not supported. Returning an empty result set.");
-    return new TimestreamSchemasResultSet();
+  public ResultSet getSchemas() throws SQLException {
+    return new TimestreamSchemasResultSet(this.connection, null);
```

Comment:
Why use 2-argument the overload, when you can use the 1-argument version?

Reply:
I don't think the 1-argument version can be used here, since the constructor `TimestreamSchemasResultSet` needs to take in 2 arguments: one for connection and another one for accepting the schema pattern